autotest/gdrivers/esric.py

```diff
@@ -119,6 +119,99 @@ def test_esric_4():
     assert cs == expectedcs, "wrong data checksum"
 
 
+###############################################################################
+# Open the tpkx dataset
+
+
+def test_tpkx_1():
+
+    gdaltest.tpkx_ds = gdal.Open("/vsizip/{data/esric/Usa.tpkx}/root.json")
+    assert gdaltest.tpkx_ds is not None, "open failed"
+
+
+###############################################################################
+# Check that the configuration was read as expected
+
+
+def test_tpkx_2():
+
+    if gdaltest.tpkx_ds is None:
+        pytest.skip()
+
+    ds = gdaltest.tpkx_ds
+    b1 = ds.GetRasterBand(1)
+
+    assert (
+        ds.RasterCount == 4 and ds.RasterXSize == 8192 and ds.RasterYSize == 8192
+    ), "wrong size or band count"
+
+    assert b1.GetOverviewCount() == 5, "Wrong number of overviews"
```

Comment:
@rouault I'm not sure what kind of zipping parameters are actually set by ArcGIS. We might probably do something, but that might not be compatible by ArcGIS, which I cannot check.

Reply:
I've removed the unnecessary files from the bundle and applied a deflate compression.

Reply:
Looking with 7z, it appears this is a zip without compression (ie it can be recompressed with ''zip -0 -r" using the infoZip utility

Reply:
It has to be uncompressed. The bundles/tiles are JPEG or PNG, so they would not compress much anyhow, not in realistic cases.

Reply:
I understand that the bundle is zipped with no compression by default for performance reasons, but now it looks like the size of the file is more important. We should anyway require to create a package from scratch by using an arcgis tool instead of trying to minify an existing package. I have no arcgis license, so I can't really proceed with this any further.

Reply:
> Though I don't see that commit in this PR

it's well included: https://github.com/OSGeo/gdal/pull/7799/commits/ad1f7e23a39449fe05f6a7cbb1dd709c2e7cd2c5